### PR TITLE
Force Transcoding for LiveTv Capability

### PR DESCRIPTION
**Pull request by Artiume**
**Changes**

Web Side PR to allow force transcoding. This should fix several issues. On Android clients, I am currently unable to stream livetv since the backend doesn't fallback to transcode on Direct Play failure. This is also an attempt to increase play-ability for clients that don't correctly report their compatibility. See attached issue for source.

**Issues**

https://github.com/jellyfin/jellyfin/issues/1202

### CODE REVIEW

File: src/strings/en-us.json [hunk 1]
```diff
@@ -1041,6 +1041,7 @@
     "OptionAlbumArtist": "Album Artist",
     "OptionAllUsers": "All users",
     "OptionAllowAudioPlaybackTranscoding": "Allow audio playback that requires transcoding",
+    "OptionForceRemoteSourceTranscoding": "Force remote media sources transcoding"
```
Comment (thornbill) [inline src/strings/en-us.json]: You are missing a comma at the end of this line.

Comment (Artiume) [reply src/strings/en-us.json]: fixed

File: src/strings/en-us.json [hunk 2]
```diff
@@ -1042,6 +1042,7 @@
     "OptionAlbumArtist": "Album Artist",
     "OptionAllUsers": "All users",
     "OptionAllowAudioPlaybackTranscoding": "Allow audio playback that requires transcoding",
+    "OptionForceRemoteSourceTranscoding": "Force remote media sources transcoding",
```
Comment (JustAMan) [inline src/strings/en-us.json]:
```suggestion
    "OptionForceRemoteSourceTranscoding": "Force transcoding of remote media sources (like LiveTV)",
```
?

**Comment by JustAMan** [comment]
Note that accompanying server PR https://github.com/jellyfin/jellyfin/pull/2160 must be merged first.

**Comment by Artiume** [comment]
Out of scope, but should we rename the Feature Access section to Live TV? I don't think JF intends to have 'features'

https://github.com/jellyfin/jellyfin-web/blob/86bf6ab8d2999398b3d1775f5f20ba659ef3fd0d/src/strings/en-us.json#L362

**Comment by dkanada** [comment]
Do you have a screenshot to explain the location and purpose? I've never seen it before.

**Comment by Artiume** [comment]
what's a good way to upload images to git? You can find it exposed under Admin > Users > any_user's_profile

**Comment by Artiume** [comment]
image

**Comment by Artiume** [comment]
I'm not sure if this'll work without the backend merged

**Comment by dkanada** [comment]
Yeah, I thought it had been merged.

**Pull request merged**